src/validation.cpp

```diff
@@ -1085,63 +1085,14 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
 
     PackageValidationState package_state;
     const unsigned int package_count = txns.size();
```

Comment:
This local variable is no longer needed:

```diff
diff --git a/src/validation.cpp b/src/validation.cpp
index fef880152a..a5d7966b09 100644
--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -1083,13 +1083,12 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
 {
     AssertLockHeld(cs_main);
 
-    PackageValidationState package_state;
-    const unsigned int package_count = txns.size();
     // These context-free checks can be done before grabbing the mempool lock.
+    PackageValidationState package_state;
     if (!CheckPackage(txns, package_state)) return PackageMempoolAcceptResult(package_state, {});
 
     std::vector<Workspace> workspaces{};
-    workspaces.reserve(package_count);
+    workspaces.reserve(txns.size());
     std::transform(txns.cbegin(), txns.cend(), std::back_inserter(workspaces),
                    [](const auto& tx) { return Workspace(tx); });
     std::map<const uint256, const MempoolAcceptResult> results;
```

Reply:
done